src/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -78,29 +80,72 @@ const OrderRow = (props: OrderRowProps) => {
   )
 }
 
+const PAGE_SIZE = 5
+
+const loadNext = (pageInfo, relay, setLoading) => {
+  const { hasNextPage, endCursor } = pageInfo
+
+  if (hasNextPage) {
+    this.loadAfter(endCursor, relay, setLoading)
+  }
+}
+
+const loadAfter = (cursor, relay, setLoading) => {
+  setLoading(true)
+
+  relay.refetch(
+    {
+      first: PAGE_SIZE,
+      after: cursor,
+      before: null,
+      last: null,
+    },
+    null,
```

Comment:
missing renderVariables

const renderVariables = {
   first: <total items of connection>
}

Reply:
🙏  @sibelius for looking at the code. Adding `renderVariables` by itself didn't fix the problem and something else should also be wrong but since we are moving that graphql endpoint I leave this PR and continue working on that when that is merged.